src/mtev_cluster.h

```diff
@@ -56,6 +56,13 @@ typedef struct {
   uint8_t number_of_payloads;
 } mtev_cluster_node_t;
 
+typedef enum {
+  mtev_cluster_node_died,
+  mtev_cluster_node_rebooted,
+  mtev_cluster_node_changed_seq,
+  mtev_cluster_node_changed_payload
+} mtev_cluster_node_changes;
```

Comment:
We typically all caps enum elements.

Also, it seems that more than one of these could be true at the same time. Would a bit set be more appropriate?